pkg/streamer/file_test.go

```diff
@@ -412,8 +431,10 @@ func (t *testUtilSuite) TestRelaySubDirUpdated(c *C) {
 	wg.Add(1)
```

Comment:
seems line 442 to 460 should be put above this line

Reply:
ditto